### PR TITLE
Minor fixes for clang compatibility.

### PR DESCRIPTION
**Pull request by cynthia**
Nothing of significant functionality, mostly relates to #6.

### CODE REVIEW

**Comment by cynthia** [comment]
Actually, fmin/fmax is functionally different than min/max and is probably for the better. While unlikely to encounter this - NaN handling behavior is better defined in fmin/fmax.

This comes with a tradeoff of older compiler compatibility.

File: src/hnsw.cc [hunk 1]
```diff
@@ -38,6 +39,8 @@ using std::endl;
 using std::fstream;
 using std::max;
 using std::min;
+using std::fmax;
+using std::fmin;
```
Comment (ummae) [inline src/hnsw.cc]: It does not needed since you use fmin/fmax with std namespace explicitly.

Comment (cynthia) [reply src/hnsw.cc]: I was actually a bit curious about this entire block. Easy fix.

**Comment by ummae** [comment]
@cynthia Thank you for your work 👍

**Pull request merged**